docs/topics/wasm/wasm-troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+[//]: # (title: Troubleshooting)
+
+Most of the browsers support WebAssembly. For recent browser versions, WebAssembly works by default. However,
```

Comment:
It would be nice to put it into a referable subsection (something about Browsers)